client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/SubmitCommand.kt

```diff
@@ -23,6 +27,7 @@ internal class SubmitCommand(private val submissionService: SubmissionService) :
     private val onBehalf by option("-b", "--onBehalf", help = ON_BEHALF_HELP)
     private val input by option("-i", "--input", help = INPUT_HELP).file(exists = true).required()
     private val attached by option("-a", "--attached", help = ATTACHED_HELP)
+    private val moveFiles by option("-m", "--moveFiles", help = MOVE_FILES).flag(default = false)
```

Comment:
I think here we should use same convention as in rest api which is fileMode with default value.  In  that way is easier if someone is transition from comeline to direct api call or when discussing about the functionality.